config/application.rb

```diff
@@ -2,20 +2,7 @@
 
 require_relative "boot"
 
-require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
-require "active_record/railtie"
-require "active_storage/engine"
-require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
-require "action_view/railtie"
-require "action_cable/engine"
-# require "sprockets/railtie"
-# require "rails/test_unit/railtie"
```

Comment:
`test_unit` is a part of `rails/all`, but I guess it is not a problem that it is loaded?

Reply:
I ran the upgrade script, it didn't complain about conflicts in application.rb.

I'll change it back so we only get the parts we need.

Reply:
Fixed in main